Element.js

```diff
@@ -287,8 +297,22 @@ Element.prototype = {
 			}
 		}
 
-		// If the input isn't a filename, just post the value directly
-		return this._post('value', getPostData(value)).then(noop);
+		function sendPostData(paramName) {
+			return self._post('value', getPostData(value, paramName)).then(noop).catch(function(error) {
+				if (error.detail.error === 'invalid argument'
+					&& paramName !== 'text') {
+					self.session.capabilities.typeParameterCalledText = true;
+					console.log(value);
```

Comment:
This `console.log` probably shouldn't be here.

Reply:
Thanks. Removed.